app/views/home/docs.html.erb

```diff
@@ -198,14 +198,35 @@
 
   <pre>
     {
-      "fedeo": {
+      "provider_name": {
         "collections": 263,
         "granules": 0,
         "last_error": null,
         "last_requested_at": "2022-07-22T14:38:02Z",
         "updated_at": "2022-07-22T14:38:04Z"
       }
     }</pre>
+
+  <p>
+    To retrieve information for a specific provider and their collections (in JSON format) visit the
+    <a target="_blank" class="ext" href="<%= Rails.configuration.relative_url_root%>/api/holdings/provider_name.json">provider holdings endpoint</a>, ensure to change `provider_name` to a support OpenSearch provider.
```

Comment:
Possible grammar change `to a supported OpenSearch provider.` Unless `support OpenSearch` is a designation.